test/unit/asrouter/MessageLoaderUtils.test.js

```diff
@@ -135,4 +136,34 @@ describe("MessageLoaderUtils", () => {
       assert.isFalse(MessageLoaderUtils.shouldProviderUpdate({id: "foo", lastUpdated: 0,  updateCycleInMs: 300}));
     });
   });
+
+  describe("#installAddonFromURL", () => {
+    let globals;
+    let sandbox;
+    let getInstallStub;
+    let installAddonStub;
+    beforeEach(() => {
+      globals = new GlobalOverrider();
+      sandbox = sinon.sandbox.create();
+      getInstallStub = sandbox.stub();
+      installAddonStub = sandbox.stub();
+      globals.set("AddonManager", {
+        getInstallForURL: getInstallStub,
+        installAddonFromWebpage: installAddonStub
+      });
+    });
+    afterEach(() => {
+      sandbox.restore();
+      globals.restore();
+    });
+    it("should call installAddonFromWebpage with correct arguments", async () => {
+      getInstallStub.resolves(null);
+      installAddonStub.resolves(null);
+
+      await MessageLoaderUtils.installAddonFromURL({}, "foo.com");
+
+      assert.calledOnce(getInstallStub);
+      assert.calledOnce(installAddonStub);
```

Comment:
you're not actually checking that these were called with the correct arguments, despite what the test is named ;)

Reply:
I will update the test description. The function is only calling apis and passing results without any transformations. Stubbing all the various functions in order to assert doesn't seem very useful.